### ISSUE TITLE
can you help me？

### ISSUE THREAD

**Issue by PhillipWangAust** [vonNiklasson/graph-dac#1]
Hello, thank you very much for the code you provided. This is very helpful for understanding the distributed average consensus, but I have some problems when running the code. Can you give me some solutions?
  File "F:/PythonSpace/graph-dac-master/average consusens/src/cmd.py", line 3, in <module>
    from Graphs import Network

ModuleNotFoundError: No module named 'Graphs'

**Comment by vonNiklasson** [comment]
Hi Phillip! Thank you for the star. This repo is part of a final bachelor thesis and is currently somewhat unorganized. We are a bit bad at using branches at the moment so the code is a bit incomplete. I'd suggest using our last working state with commit `b5db558` which should contain a working example of our code.

Since our repo has gained more public interest than we anticipated we'll likely structure it with branches and releases in the future.

Please let me know if you have any further questions or problems.

**Comment by PhillipWangAust** [comment]
Thank you very much for your answer. I look forward to further achievements in your research work.

**Comment by PhillipWangAust** [comment]
Hi, maybe the version you are using python is a bit confusing, there are some minor problems, I have already labeled it, and if you have time, you can correct it. thank you

  File "F:\PythonSpace\graph-dac-master\average consusens\src\GraphConverter\GraphConverter.py", line 29
    print origin, destination
               ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(origin, destination)?

**Comment by vonNiklasson** [comment]
Yes, the repo is currently designed for Python 2.7, but there's not really any reason not to convert it to 3.x. Depending on the new packages that's going to be used, we'll probably make it work with 3.x later on.

**Comment by vonNiklasson** [comment]
Hello again @PhillipWangAust, I have made a release of the current working state of the repository, mainly [Release 0.1](https://github.com/vonNiklasson/graph-dac/releases/tag/0.1). It will preferably be better to [browse and fetch code by the release tags](https://github.com/vonNiklasson/graph-dac/tree/0.1) instead of commits since it's prone to change rapidly the coming weeks.

**Comment by PhillipWangAust** [comment]
Yes, I saw these changes. Thank you very much for all your hard work. Your work is very helpful in understanding the distributed average consensus.
